### PR TITLE
Use apache mod classes

### PR DESCRIPTION
**Pull request by ekohl**
Requires https://github.com/puppetlabs/puppetlabs-apache/pull/1588

### CODE REVIEW

**Comment by ekohl** [comment]
The pl-apache PR has been merged and rebased. Do we want to wait for a pl-apache release though?

File: manifests/config/passenger.pp [hunk 1]
```diff
@@ -95,6 +95,13 @@
   include ::apache::mod::headers
   include ::apache::mod::passenger
 
+  if $::foreman::ipa_authentication {
```
Comment (domcleal) [inline manifests/config/passenger.pp]: This should be added to the local class parameters list so it can be set in foreman::config::passenger tests.

Comment (ekohl) [inline manifests/config/passenger.pp]: Updated with tests

File: spec/classes/foreman_config_passenger_spec.rb [hunk 2]
```diff
@@ -314,6 +319,41 @@
           should contain_apache__vhost('foreman-ssl').with_passenger_pre_start("https://#{facts[:fqdn]}:8443")
         end
       end
+
+      describe 'with different ports set' do
```
Comment (domcleal) [inline spec/classes/foreman_config_passenger_spec.rb]: `with ipa_authentication => true` ?

Comment (ekohl) [inline spec/classes/foreman_config_passenger_spec.rb]: It's still early. Fixed

**Comment by ekohl** [comment]
Rebased now that puppetlabs-apache 2.0.0 is out.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**